data-model-for-portal/Microsoft.Compute/virtualMachines.json

```diff
@@ -167,6 +243,164 @@
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "display VM's managed identity info.",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--name",
+                            "value": "[path.vmName]"
+                        },
+                        {
+                            "name": "--resource-group",
+                            "value": "[path.resourceGroupName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az vm encryption show",
+            "description": "Show encryption status.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}?$expand=instanceView",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/vm/encryption#az-vm-encryption-show"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "[--ids]",
+                            "[--name]",
+                            "[--resource-group]",
+                            "[--subscription]"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "Show encryption status.",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--name",
+                            "value": "[path.vmName]"
+                        },
+                        {
+                            "name": "--resource-group",
+                            "value": "[path.resourceGroupName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az vm unmanaged-disk list",
+            "description": "List unmanaged disks of a VM.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/vm/unmanaged-disk#az-vm-unmanaged-disk-list"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "--name",
```

Comment:
- Miss [] for parameters
- Use `--vm-name` instead of `--name`

Reply:
Thank you for your helpful suggestion. Already modified. Ready for rereview.